Assets/HoloToolkit/Utilities/Scripts/SceneLauncher.cs

```diff
@@ -39,6 +39,20 @@ private void OnValidate()
             Debug.Assert(SceneButtonPrefab != null, "SceneLauncher.SceneButtonPrefab is not set.");
         }
 
+        protected override void Awake()
+        {
+            // If we have already initialized,
+            // then we've created a second one.
+            if (IsInitialized)
```

Comment:
I like how you're using Singleton.IsInitialized to gate destruction of the redundant DontDestoyOnLoad objects.

Reply:
😄